articles/appliance/webtasks.md

```diff
@@ -27,6 +34,92 @@ The `auth0-sandbox` is the recommended method for running your custom code.
 
 You may use Webtasks by calling its endpoints directly. This can be done using the Webtask Command Line Interface (`wt-cli`) and specifying the ``--url "https://webtask.<a0url>.com"`` parameter (where `a0url` is the address of the Appliance node). For additional information on setting up the `wt-cli`, please see [Getting Started with Webtasks](https://webtask.io/docs/101).
 
+### Node.js Modules
+
+Currently, not all of the [Node.js modules available for the Auth0 Cloud Environment](https://tehsis.github.io/webtaskio-canirequire/) are available for the Appliance.
+
+To see which modules are available for Webtasks running on Appliance instances, execute the [`List Modules` Webtask](https://github.com/tehsis/webtaskio-canirequire/blob/gh-pages/tasks/list_modules.js) using the appropriate sandbox on your Appliance instance.
+
+First, copy locally the [`List Modules` Webtask](https://github.com/tehsis/webtaskio-canirequire/blob/gh-pages/tasks/list_modules.js), either by downloading the file or by copying this code:
+
+```js
+'use npm';
+const Fs = require('fs');
+const Verquire = require('verquire');
+const _ = require('lodash@4.8.2');
+
+const abcsort = function (a, b) {
+  if (a.name < b.name) {
+      return -1;
+  }
+
+  if (a.name > b.name) {
+      return 1;
+  }
+
+  return 0;
+};
+
+
+const natives = Object.keys(process.binding("natives"))
+    .filter(nativeDep => nativeDep[0] !== '_')
+    .map(dep => ({name: dep, version: 'native'}))
+    .sort(abcsort);
+
+
+const modules = _.flatMap(Verquire.modules, (versions, module_name) => {
+   return versions.map((version) => {
+       const moduleObj = {
+         name: module_name,
+         version: version
+       };
+
+       return moduleObj;
+   });
+}).sort(abcsort);
+
+module.exports = cb => {
+    cb(null, {
+        node_version: process.version,
+        modules: natives.concat(modules)
+    });
+};
+```
+
+Afterwards, create a webtask profile using `wt-cli`, if you don't already have one.
+
+```bash
+wt init --container "YOUR_TENANT_NAME" --url "https://sandbox.it.auth0.com" --token "eyJhbGci..." -p "a``YOUR_TENANT_NAME-default"
+```
+
+::: panel-info wt init
+This is the very first command you will run once you install `wt-cli`. It is used to create and update webtask profiles. It also provisions a webtask token if you don't have one already. Each webtask profile has a name, a URL and a container. In order to configure a webtask profile you need to provide an email or phone number. You will receive a verification code there which you will have to input to the terminal. For more details you can use the wt built-in help: `wt init -h`.
```

Comment:
@tehsis Does this process (`You will receive a verification code...`) stays the same for appliance users or should I remove this?